xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -1473,5 +1473,55 @@ test_build:
         </listitem>
       </itemizedlist>
     </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.use_cases.config_url_and_placeholder_variables">
+      <title>Using Custom Configuration File URL with Placeholder Variables</title>
+
+      <para>It may happen that you have multiple repositories following the same
+      set of steps, and so you would rather have one copy of the configuration
+      file applied to multiple repositories. The combination of placeholder
+      variables with configuration file url setting lets you do just that.</para>
+
+      <para>Let's say you have the following configuration file that works with
+      your SCM repository called <emphasis>gnome-shell</emphasis></para>
+
+      <screen>workflow:
+  steps:
+    - branch_package:
+        source_project: "test-project"
+        source_package: "gnome-shell"
+        target_project: "test-target-project"
+  filters:
+    event: pull_request</screen>
+
+      <para>If you replaced <emphasis>gnome-shell</emphasis> with
+      <emphasis>%{SCM_REPOSITORY_NAME}</emphasis> like so:</para>
+
+      <screen>workflow:
+  steps:
+    - branch_package:
+        source_project: "test-project"
+        source_package: "%{SCM_REPOSITORY_NAME}"
+        target_project: "test-target-project"
+  filters:
+    event: pull_request</screen>
+
+      <para>It should still perform just as well as it did before, however now
+      this configuration can be applied to any other OBS package in the
+      <emphasis>test-project</emphasis> and SCM repository combination assuming
+      they have the same name.</para>
+
+      <para>From here the only thing left to do would be to host this file
+      somewhere where OBS can access it, creating a workflow token and a
+      corresponding webhook (following the <link linkend="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication">setup instructions</link>)
+      for every repository you want this configuration file to apply to,
+      making sure you <link linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.config_location">set the correct configuration url</link>.</para>
+
+      <para>There are many other ways to use these two features in parallel,
+      make sure to read documentation about both the
+      <link linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.placeholder_variables">placeholder variables</link>
+      as well as <link linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.config_location">custom configuration paths and urls</link>
```

Comment:
Consider using `<xref linkend="..."/>` instead of `<link>...</link>`. In 99% of the cases this is what we use in our product documentation. :slightly_smiling_face: 

With xref, the tag is being resolved automatically and creates a label (usually "section", "chapter" etc.), an optional number, and the title in quotes. This is very helpful as the resolved text is always correct. With `<link>` it's only correct as long as you don't change the text inside the link tag. You don't want to adapt all the text for all link tags, do you? :wink: 

If you use xref, perhaps you need to adapt the surrounding text a bit so it fits into the description. With the little help of the [xrefstyle attribute](http://www.sagehill.net/docbookxsl/CustomXrefs.html#UsingSelect) you can adapt the output even more.

For example, the [ALP documentation](https://documentation.suse.com/alp/all/single-html/alp/#related-alp-deployment) contains this text:

```
For deploying ALP with D-Installer, refer to Section 2.5, “Deploying ALP using D-Installer”.
```

The text `Section 2.5, “Deploying ALP using D-Installer”` is created from an `<xref/>`.